src/main/java/com/textmessenger/service/PostServiceImpl.java

```diff
@@ -1,34 +1,68 @@
 package com.textmessenger.service;
 
-import com.textmessenger.constant.NotificationType;
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.model.ObjectMetadata;
+import com.textmessenger.config.AmazonConfig;
 import com.textmessenger.model.entity.Post;
 import com.textmessenger.model.entity.User;
 import com.textmessenger.model.entity.dto.PostToFront;
 import com.textmessenger.repository.PostRepository;
+import com.textmessenger.repository.UserRepository;
+import com.textmessenger.security.UserPrincipal;
 import org.springframework.data.domain.Sort;
+import org.springframework.security.core.context.SecurityContextHolder;
 import org.springframework.stereotype.Service;
 import org.springframework.transaction.annotation.Transactional;
+import org.springframework.web.multipart.MultipartFile;
 
+import java.io.IOException;
+import java.io.InputStream;
 import java.util.List;
+import java.util.UUID;
 
 @Service
 @Transactional
 public class PostServiceImpl implements PostService {
-
+  final String bucket = AmazonConfig.BUCKET_NAME;//NOSONAR
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. ([squid:S00115](https://sonarcloud.io/organizations/final_project_text_messenger/rules#rule_key=squid%3AS00115))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=Text_Messenger_Project&pullRequest=160&issues=AWWasFh23Qu1svVwx8tI&open=AWWasFh23Qu1svVwx8tI)

[ISSUE_KEY]: AWWasFh23Qu1svVwx8tI